### PR TITLE
1386-Reduce-Character-Limits

### PR DESCRIPTION
**Pull request by Dmac26**
﻿## Summary
Addresses Issue #1386  

This code update reduces the character limit on the "Business name" and "Organization name" fields. Additionally hint text was added to warn the user that the character restriction is in place and an error message is thrown if the user exceeds those character limits.

## This pull request is ready to merge when...
- [x] Feature branch starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [x] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [ ] This code has been reviewed by someone other than the original author﻿##

### CODE REVIEW

File: frontend/src/app/application-forms/fields/organization-name.component.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <ng-container [formGroup]="applicantInfo">
   <label id="organization-name-label" class="usa-input">{{ name }} <span class="required-fields-asterisk">*</span></label>
+  <p id="organization-name-hint-text" class="help-text usa-form-hint">NOTE: The {{ name }} field only allows 30 characters for input.</p>
```
Comment (briandavidson) [inline frontend/src/app/application-forms/fields/organization-name.component.html:3]: @Dmac26 per our conversation off github, even though it's a small addition we should check with @aQuib and get his approval for any changes that the user will see.

What'cha think @aQuib, I'm guessing this is g2g?

**Comment by mgwalker** [comment]
I'm late to the party on this one, but in the future, what do y'all think about adding tests to cover these kinds of scenarios? Since it's a known issue with a downstream service, having a test to verify that invalid data won't be accepted would be

man doing the "chef kissing fingers" gesture

**Comment by briandavidson** [comment]
> I'm late to the party on this one, but in the future, what do y'all think about adding tests to cover these kinds of scenarios? Since it's a known issue with a downstream service, having a test to verify that invalid data won't be accepted would be

I think that's a great idea, agree!

**Comment by Dmac26** [comment]
@mgwalker @briandavidson - Yeah, that is a good idea. I will put together a card for that and get a jump on.

**Pull request merged**